### ISSUE TITLE
[4.0] Clean install broken!

### ISSUE THREAD

**Issue by infograf768** [joomla/joomla-cms#24300]
### Steps to reproduce the issue
Install 4.0_dev branch
Load admin.

```
An error has occurred.

    0 Call to a member function addScript() on null 
```

**Comment by ghost** [comment]
Confirmed (install on Cloudaccess).

**Comment by SharkyKZ** [comment]
Side effect of #24272.

**Comment by infograf768** [comment]
@SharkyKZ 
and... ?

**Comment by SharkyKZ** [comment]
Working on it.

**Comment by SharkyKZ** [comment]
FWIW, I've been meaning to open an issue about document not being available early enough in the app today.

**Comment by SharkyKZ** [comment]
Test PR #24303 please.

**Comment by ghost** [comment]
closed as having Pull Request #24303
